frontend/src/cards/UnknownsMapCard.tsx

```diff
@@ -185,9 +185,14 @@ function UnknownsMapCardWithKey(props: UnknownsMapCardProps) {
         const unknownsAllZero =
           unknowns.length > 0 &&
           unknowns.every(
-            (unknown: HetRow) =>
-              unknown[metricConfig.metricId] === 0 ||
-              unknown[metricConfig.metricId] == null,
+            (unknown: HetRow) => unknown[metricConfig.metricId] === 0,
+          )
+
+        // for data sets where some geos might contain null
+        const unknownsAllNull =
+          unknowns.length > 0 &&
+          unknowns.every(
+            (unknown: HetRow) => unknown[metricConfig.metricId] === null,
```

Comment:
i think this variable is unused; probably can remove this chunk of code.

Reply:
unless we want to use it to prevent an entirely gray "unknowns map" from rendering which might be good?

Reply:
Should we keep the variable for consistency with other checks and integrate it into visualization logic, or remove it as unused code? I wonder if there is an instance at the geo state level for an entirely gray "unknowns map"?

Reply:
let's just solve one problem at a time (the no data label) so remove this chunk of code since it's not being used. if we decide later to add a feature when blank maps are suppressed we can pursue it at that time